Sally7/ValueConversion/ToS7Conversions.cs

```diff
@@ -189,5 +180,46 @@ private static int ConvertFromString(string? value, int length, Span<byte> outpu
             return span.Length + 2;
 #endif
         }
+
+        private static int CopyBuffer(ReadOnlySpan<byte> input, Span<byte> output)
+        {
+            ref var destination = ref MemoryMarshal.GetReference(output);
+            ref var source = ref MemoryMarshal.GetReference(input);
+
+            var offset = 0u;
+            while (offset + sizeof(ulong) <= input.Length)
+            {
+                var value = Unsafe.ReadUnaligned<ulong>(ref source.GetOffset(offset));
+                NetworkOrderSerializer.WriteUInt64(ref destination.GetOffset(offset), value);
+
+                offset += sizeof(ulong);
+            }
+
+            if (offset + sizeof(uint) <= input.Length)
```

Comment:
Here it doesn't really matter, but I'd write it as
```suggestion
            if (offset <= input.Length - sizeof(uint))
```
to be consistent with the loop above.

Same for the other checks below.